src/one_dragon_qt/widgets/download_card/launcher_download_card.py

```
@@ -0,0 +1,315 @@
+        backup_path = Path(work_dir) / LAUNCHER_BACKUP_NAME
+
+        if backup_path.exists():
+            try:
+                backup_path.unlink()
+                log.info(f'删除备份文件: {LAUNCHER_BACKUP_NAME}')
+            except Exception as e:
+                log.error(f'删除备份文件失败 {LAUNCHER_BACKUP_NAME}: {e}')
+
+    def _on_download_finish(self, success: bool, message: str) -> None:
+        """
+        下载完成后处理备份：成功则删除备份，失败则回滚
```

Ruff (0.14.1)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)